docs/sources/setup-grafana/restart-grafana.md

```diff
@@ -1,102 +1,102 @@
+title: Start the Grafana server
+menuTitle: Start Grafana
 weight: 300
 ---
 
-# Restart Grafana
+# Start the Grafana server
 
-Users often need to restart Grafana after they have made configuration changes. This topic provides detailed instructions on how to restart Grafana supported operating systems.
+This topic includes instructions for starting the Grafana server. For certain configuration changes, you might have to restart the Grafana server for them to take effect.
 
-- [Windows](#windows)
-- [MacOS](#macos)
-- [Linux](#linux)
-- [Docker](#docker)
+The following instructions start the `grafana-server` process as the `grafana` user, which was created during the package installation.
 
-## Windows
+If you installed with the APT repository or `.deb` package, then you can start the server using `systemd` or `init.d`. If you installed a binary `.tar.gz` file, then you execute the binary.
 
-To restart Grafana:
+## Linux
 
-1. Open the Services app.
-1. Right-click on the **Grafana** service.
-1. In the context menu, click **Restart**.
+Starting and restarting the Grafana server depends on whether your Linux system uses `systemd` or `init.d`.
 
-## macOS
+### Start the Grafana server with systemd
 
-Restart methods differ depending on whether you installed Grafana using Homebrew or as standalone macOS binaries.
+Complete the following steps to start the Grafana server with systemd and verify that it is running:
 
-### Restart Grafana using Homebrew
+1. To start the service, run the following commands:
 
-Use the [Homebrew](http://brew.sh/) restart command:
+   ```bash
+   sudo systemctl daemon-reload
+   sudo systemctl start grafana-server
+   sudo systemctl status grafana-server
+   ```
 
-```bash
-brew services restart grafana
-```
+1. To verify that the service is running, run the following command:
 
-### Restart standalone macOS binaries
+   ```
+   Need input here
+   ```
 
-To restart Grafana:
+1. To configure the Grafana server to start at boot, run the following command:
 
-1. Open a terminal and go to the directory where you copied the install setup files.
-1. Run the command:
+   ```bash
+   sudo systemctl enable grafana-server.service
+   ```
 
-```bash
-./bin/grafana-server web
-```
+#### Restart the server
 
-## Linux
+Complete the following steps to restart the Grafana server:
 
-Restart methods differ depending on whether your Linux system uses `systemd` or `init.d`.
+1. To restart the Grafana server, run the following commands:
 
-### Restart the server with systemd
+   ```bash
+   sudo systemctl restart grafana-server
+   sudo systemctl status grafana-server
+   ```
 
-To restart the service and verify that the service has started, run the following commands:
+1. To configure the Grafana server to restart at boot, run the following command:
 
-```bash
-sudo systemctl restart grafana-server
-sudo systemctl status grafana-server
-```
+   ```bash
+   sudo systemctl enable grafana-server.service
+   ```
```

Comment:
this is not relevant or correct, setting grafana to start at boot is a one-time install task

Reply:
leaving here as discussed.